vignettes/shiny.Rmd

```diff
@@ -107,7 +107,7 @@ svg.selectAll('rect')
     .attr('fill', 'steelblue')
     .attr("d", function(d) { return d; })
     .on("click", function(){
-      Shiny.onInputChange(
+      Shiny.setInputValue(
```

Comment:
Show mentioned that there was another flag that we should pass to `setInputValue`, let's do that as well.